config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -132,10 +132,11 @@ postsubmits:
       containers:
       - image: gcr.io/k8s-testimages/gcloud-bazel:v20210128-v0.6-11-gf1c6399
```

Comment:
Does this image have `make` and `kubectl`?

Reply:
yeah it's a ubuntu image with kubectl installed https://github.com/bazelbuild/rules_k8s/blob/d05cbea5c56738ef02c667c10951294928a1d64a/images/gcloud-bazel/Dockerfile#L58

I'll unhold tomorrow morning